### ISSUE TITLE
FAQ : Does "InternVL/internvl_chat/internvl/train/internvl_chat_finetune.py" support InternVL3.5 training ?

### ISSUE THREAD

**Issue by HuangChiEn** [OpenGVLab/InternVL#1192]
I had saw someone applied "InternVL/internvl_chat/internvl/train/internvl_chat_finetune.py" to finetune InternVL3.5 and but i want to confirm is it a official way to do that ?

check list :
1.  Conversation template is "internvl**2_5**", is it valid for InternVL**3.5** ?
2.  Preprocess function is "internvl**2_5**", is it valid for InternVL**3.5** ?
<img width="1023" height="516" alt="Image" src="https://github.com/user-attachments/assets/b1adbee9-0026-4396-9e7a-c5bd344d9c1d" />

Any suggestion may be helpful ~

**Comment by wuzhaodongaipython** [comment]
我也需要微调3.5，我感觉是不是得找sh文件？是哪个sh文件呢，你有找到不

**Comment by CrazyElements** [comment]
Good questions. I see InternVL3.0's conv_type is also internvl2_5, so i guess conv_type = internvl2_5 is general?  And btw, in internvl_chat_gpt_oss/shell there exists scripts for internvl3_5 fine-tuning. Hope this could help you.

**Comment by HuangChiEn** [comment]
> Good questions. I see InternVL3.0's conv_type is also internvl2_5, so i guess conv_type = internvl2_5 is general? And btw, in internvl_chat_gpt_oss/shell there exists scripts for internvl3_5 fine-tuning. Hope this could help you.

> 我也需要微调3.5，我感觉是不是得找sh文件？是哪个sh文件呢，你有找到不

如同[CrazyElements](https://github.com/CrazyElements)所說，目前我也是用 `conv_type = internvl2_5`，但微調完後3.5後期爆nan loss，雖然也有可能是我的資料集或multiple loss的影響，至少它應該是可以train ~

至於sh，**我是fork其他專案**，他們自己有實現資料載入，img token替換等，所以很遺憾我只要找到 conv_type的對應，剩下的整合就在專案內完成，所以我沒有使用到internvl啟動訓練的腳本

**Comment by HuangChiEn** [comment]
Actually there's one way, but now i'm too lazy to do so ~
Officially, internVL3.5 apply `apply_chat_template`, which do the same mechanism as **Conversation class**, so we can inspect the [jinja template](https://huggingface.co/OpenGVLab/InternVL3_5-1B/blob/main/chat_template.jinja), and make a equivalent routine in Conversation class will be fine ~
